src/db/models.py

```diff
@@ -122,3 +127,45 @@ class BoardLabelsFilterAssociation(Base):
 
     board_id = Column(Integer, ForeignKey("boards.id"), primary_key=True)
     label_id = Column(Integer, ForeignKey("labels.id"), primary_key=True)
+
+
+table_name_to_model = {
+    model.__tablename__: model
+    for model in Base.__subclasses__()
+}
```

Comment:
imo this should be done in init of models manager or be a cached property.
you rely on import order here

Reply:
moved to be method of Base